packages/connector-google/README.md

```diff
@@ -56,3 +82,60 @@ Now you should have the Google OAuth 2.0 consent screen configured.
 
 ## References
 * [Google Identity: Setting up OAuth 2.0](https://developers.google.com/identity/protocols/oauth2/openid-connect#appsetup)
+
+# Google 链接器
+
+## 在 Google API 控制台中创建一个项目
+
+- 用你的 Google 帐号登录 [Google 云端平台](https://console.developers.google.com)。
+- 点按顶部菜单栏上的「选择项目」，然后点击「新建项目」按钮，新建一个项目。
+- 在新创建的项目中，点击「API 和服务」进入「API 和服务」的页面中。
+
+## 配置 OAuth 同意屏幕
+
+### 注册和配置应用
+
+- 在左侧的「API 和服务」的菜单栏中, 点按「OAuth 同意屏幕」。
+- 选择你所需要的「User Type」，之后点击「创建」。（注意: 如果你将「User Type」设置为「外部」，在后续的步骤中你需要设置测试用户。）
+
+现在你将位于「修改应用注册」页面。
+
+### 修改应用注册
+
+#### 设置「OAuth 同意屏幕」
+
+- 根据页面提示，填写「OAuth 同意屏幕」相关表单。
+- 点按「保存并继续」以进行后续操作。
+
+#### 配置权限「范围」
+
+- 点击「添加或移除范围」按钮，然后在弹出的抽屉页面中选择 `../auth/userinfo.email`、 `../auth/userinfo.profile` 和 `openid`，然后点击「更新」以完成配置。
+- 根据你的需求填写页面上的相关的表单。
+- 点按「保存并继续」以进行后续操作。
+
+#### 添加「测试用户」（仅「User Type」设置为「外部」时需要）
+
+- 点击 「+ ADD USERS」添加测试用户，以遍这些用户能在应用的发布状态为”测试时“能访问该应用。
```

Comment:
Does it have Chinese translation for this `+ ADD USERS` button?

Reply:
Yes

Reply:
<img width="579" alt="image" src="https://user-images.githubusercontent.com/10806653/178913305-327a5cbf-0d38-4a2e-8814-4e3845023cd4.png">